.github/workflows/ci_cd.yml

```diff
@@ -61,6 +61,16 @@ jobs:
           operating-system: ${{ matrix.os }}
           python-version: ${{ matrix.python-version }}
 
+  docker-style:
+    name: Docker Style Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: PyAnsys documentation style checks
+        uses: ansys/actions/docker-style@v5
+        with:
+          directory: docker
+          recursive: true
+
```

Comment:
Why not just activate it through the code-style checks now? :)

Reply:
Agh damn you are right (as always) - I got rid of the code-style check the moment I wnt public.